### ISSUE TITLE
Samsung's One UI puts a big white bar below the keyboard, even with everything darkmoded.

### ISSUE THREAD

**Issue by ivnvitx** [AnySoftKeyboard/AnySoftKeyboard#2610]
Using One UI 2.5 with gestures, AnySoft 1.10

**Comment by skelerang** [comment]
Enabling draw under navbar in effects menu should fix the colour. 

However when I tried it with with the gesture navbar, the keyboard moves too low, overlapping with other buttons.
![Screenshot_20210121-164328_Firefox](https://user-images.githubusercontent.com/66796269/105367132-cc2e1d00-5c08-11eb-8d7b-47d1581c92a7.jpg)

**Comment by ivangjxyz** [comment]
Yes! It works really nice, now I just hope the curved edges die soon 😅
Thank you!
